src/Microsoft.IdentityModel.Tokens/SymmetricSignatureProvider.cs

```diff
@@ -168,7 +168,7 @@ public override byte[] Sign(byte[] input)
                 throw LogHelper.LogException<ObjectDisposedException>(GetType().ToString());
 
             if (_keyedHash == null)
-                throw LogHelper.LogException<InvalidOperationException>(LogMessages.IDX10623);
+                throw LogHelper.LogException<ArgumentNullException>(LogMessages.IDX10623);
```

Comment:
Is _keyedHash a parameter? Should this really be an ArgumentNullException? _keyedHash looks like a member var, not an argument to the method.